app/controllers/practices_controller.rb

```diff
@@ -529,6 +534,34 @@ def destroy_diffusion_history
     end
   end
 
+  def extend_editor_session_time
+    @practice = set_practice if @practice.blank?
```

Comment:
we can just add this to the `set_practice` before_action hook

Reply:
When I make an ajax call... the before_actions don't get run.